docs/data_sources/sensu_cluster_role.md

```diff
@@ -0,0 +1,31 @@
+# sensu_cluster_role
+
+Get information about a Sensu Cluser Role.
+
+For full documentation on Sensu Cluster Roles, see [here](https://docs.sensu.io/sensu-go/5.20/reference/rbac/#roles-and-cluster-roles).
+
+## Basic Example
+
+```hcl
+data "sensu_cluster_role" "cluster_role_1" {
+  name = "my_role"
+}
+```
+
+## Argument Reference
+
+* `name` - *Required* - The name of the Sensu cluster role.
+
+## Attribute Reference
+
+* `rule` - *Required* - See the [Sensu rbac reference](https://docs.sensu.io/sensu-go/latest/reference/rbac/#rule-attributes).
+
+### rule
+
+The `rule` block supports:
+
+* `verbs` - *Required* - See the [Sensu rbac reference](https://docs.sensu.io/sensu-go/latest/reference/rbac/#rule-attributes).
+
+* `resources` - *Required* - See the [Sensu rbac reference](https://docs.sensu.io/sensu-go/latest/reference/rbac/#rule-attributes).
```

Comment:
same here, since this is part of rule.

Reply:
hi @jtopjian 
Thank you for your review.

Unfortunately, I would like to close this PR and open it from my company git account.
With this new PR I'll address all your comments except "subject", so we can continue the discussion on the thread.

Sorry for the inconvenience.

Reply:
@fgouteroux Sure, sounds good. It's no inconvenience at all 🙂